package.json

```diff
@@ -23,8 +23,11 @@
     "eslint-plugin-import": "^2.16.0"
   },
   "optionalDependencies": {
+    "eslint-config-airbnb": "^17.1.0",
     "eslint-config-react-app": "^3.0.8",
```

Comment:
As discussed, I think we should have only 1 guideline so it's easier for new members to read up and follow.

Reply:
done - gutted react-app